### DISCUSSION TITLE
Deploying to Azure Static Web Site

### DISCUSSION THREAD

**Discussion by athelene** [quasarframework/quasar#14588] [General - Components / Directives / etc]
I would like to host my Quasar app on an Azure static web site. I see that Azure supports deploying from Github and will build the app on the Vue framework. Is there a way to do so using Quasar? Would the Vue option work? Or do I need to find another way to deploy to Azure just the files in the dist folder after building on my local machine?

**Comment by MilosPaunovic** [comment]
I think you're looking for something like this: https://github.com/marketplace/actions/azure-static-web-apps-deploy

**Comment by athelene** [comment]
Thank you.  I'll take a look at this.
